<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerDrrImageComputationPlastimatchParametersWidget</class>
 <widget class="QWidget" name="qSlicerDrrImageComputationPlastimatchParametersWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plastimatch DRR Parameters</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="ctkCollapsibleButton" name="CollapsibleButton_PlastimatchDrrImageProcessing">
     <property name="text">
      <string>Plastimatch DRR image processing</string>
     </property>
     <property name="collapsed">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="1" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="GroupBox_Intensity">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="title">
           <string>Intensity</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="1" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Range:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="ctkRangeWidget" name="RangeWidget_IntensityRange">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="decimals">
              <number>2</number>
             </property>
             <property name="minimum">
              <double>-32768.000000000000000</double>
             </property>
             <property name="maximum">
              <double>32767.000000000000000</double>
             </property>
             <property name="maximumValue">
              <double>255.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QCheckBox" name="CheckBox_InvertIntensity">
             <property name="text">
              <string>Invert</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QCheckBox" name="CheckBox_AutoscaleIntensity">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Autoscale</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="1">
           <widget class="ctkComboBox" name="ComboBox_ReconstructionAlgorithm">
            <property name="currentIndex">
             <number>1</number>
            </property>
            <item>
             <property name="text">
              <string>Exact</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Uniform</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Reconstruction algorithm:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Hounsfield units conversion:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Threading:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="ctkComboBox" name="ComboBox_HounsfieldConversion">
            <item>
             <property name="text">
              <string>Preprocess</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Inline</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>None</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="ctkComboBox" name="ComboBox_Threading">
            <item>
             <property name="text">
              <string>CPU</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>CUDA</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>OpenCL</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Hounsfield units threshold:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="ctkSliderWidget" name="SliderWidget_HounsfieldThreshold">
            <property name="toolTip">
             <string>Process HU values higher than theshold</string>
            </property>
            <property name="decimals">
             <number>0</number>
            </property>
            <property name="minimum">
             <double>-1000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>3095.000000000000000</double>
            </property>
            <property name="value">
             <double>-1000.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="CheckBox_UseExponentialMapping">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Use exponential mapping</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="ctkCollapsibleButton" name="CollapsibleButton_PlastimatchDrrArguments">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Plastimatch DRR command arguments (read only)</string>
     </property>
     <property name="collapsed">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPlainTextEdit" name="plainTextEdit_PlastimatchDrrArguments">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="ctkCollapsibleButton" name="CollapsibleButton_PlastimatchDrrMatrices">
     <property name="text">
      <string>Plastimatch DRR projection matrices (read only)</string>
     </property>
     <property name="collapsed">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Intrinsic:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Projection:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Extrinsic:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="ctkMatrixWidget" name="MatrixWidget_Extrinsic">
          <property name="minimum">
           <double>-10000.000000000000000</double>
          </property>
          <property name="maximum">
           <double>10000.000000000000000</double>
          </property>
          <property name="decimals">
           <number>4</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="ctkMatrixWidget" name="MatrixWidget_Intrinsic">
          <property name="rowCount">
           <number>4</number>
          </property>
          <property name="minimum">
           <double>-10000.000000000000000</double>
          </property>
          <property name="maximum">
           <double>10000.000000000000000</double>
          </property>
          <property name="decimals">
           <number>4</number>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="ctkMatrixWidget" name="MatrixWidget_Projection">
          <property name="rowCount">
           <number>4</number>
          </property>
          <property name="minimum">
           <double>-10000.000000000000000</double>
          </property>
          <property name="maximum">
           <double>10000.000000000000000</double>
          </property>
          <property name="decimals">
           <number>4</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkComboBox</class>
   <extends>QComboBox</extends>
   <header>ctkComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>ctkMatrixWidget</class>
   <extends>QWidget</extends>
   <header>ctkMatrixWidget.h</header>
  </customwidget>
  <customwidget>
   <class>ctkRangeWidget</class>
   <extends>QWidget</extends>
   <header>ctkRangeWidget.h</header>
  </customwidget>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header>ctkSliderWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
